core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCatalog.java

```diff
@@ -105,23 +104,25 @@ public void testReplaceTxnBuilder() throws Exception {
     createTxn.commitTransaction();
 
     Table table = catalog.loadTable(tableIdent);
-    Assert.assertNotNull(table.currentSnapshot());
+    Assertions.assertThat(table.currentSnapshot()).isNotNull();
 
     Transaction replaceTxn =
         catalog.buildTable(tableIdent, SCHEMA).withProperty("key2", "value2").replaceTransaction();
     replaceTxn.commitTransaction();
 
     table = catalog.loadTable(tableIdent);
-    Assert.assertNull(table.currentSnapshot());
+    Assertions.assertThat(table.currentSnapshot()).isNull();
     PartitionSpec v1Expected =
         PartitionSpec.builderFor(table.schema())
             .alwaysNull("data", "data_bucket")
             .withSpecId(1)
             .build();
-    Assert.assertEquals("Table should have a spec with one void field", v1Expected, table.spec());
+    Assertions.assertThat(table.spec())
+        .as("Table should have a spec with one void field")
+        .isEqualTo(v1Expected);
 
-    Assert.assertEquals("value1", table.properties().get("key1"));
-    Assert.assertEquals("value2", table.properties().get("key2"));
+    Assertions.assertThat(table.properties()).containsEntry("key1", "value1");
+    Assertions.assertThat(table.properties()).containsEntry("key2", "value2");
```

Comment:
can be chained together

Reply:
Based on your review, I've modified the code to be more concise.